.bummr-build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+bundle exec rake
```

Comment:
'bundle exec rspec' to match what travis runs? (also probably don't want to rely on default rake target)